gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/process/traversal/util/BytecodeHelper.java

```diff
@@ -350,6 +351,26 @@ public static <A extends TraversalStrategy> Iterator<A> findStrategies(final Byt
                 os -> (A) os.getArguments()[0]);
     }
 
+    public static <A extends TraversalStrategy> boolean removeStrategies(final Bytecode bytecode, final String operator, final Class<TraversalStrategy>[] clazzes) {
```

Comment:
Is `<A extends TraversalStrategy>`  needed? `A` doesn't seem to be used.

Reply:
Thanks for noticing. Removed.